dev/io.openliberty.pages.4.0.internal.factories/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,116 @@
+#Ant properties
```

Comment:
Are the settings here consistent with the other EE11 settings that we updated previously?